src/sidebar.js

```diff
@@ -209,6 +202,36 @@ class Sidebar extends React.Component {
 
     let rootProps = {
       style: styles.root,
+    };
+
+    rootProps.style.top = this.props.topSidebar;
+
+    if (this.props.pullRight) {
+      if (this.props.enableShadow) {
+        sidebarStyle.boxShadow = '-2px 2px 4px rgba(0, 0, 0, 0.15)';
+      }
+      sidebarStyle = update(sidebarStyle, { $merge: {
+        right: 0,
+        transform: 'translateX(100%)',
+        WebkitTransform: 'translateX(100%)',
+      }});
+
+      contentStyle.transition = 'right .3s ease-out';
+
+      styles.dragHandle.left = 0;
```

Comment:
Please do not mutate constants in the top of the file but use the `update` method to create a new object.